sdk/schemaregistry/azure-schemaregistry-avroserializer/setup.py

```diff
@@ -40,7 +40,7 @@
         'azure.schemaregistry',
     ]
 install_packages = [
-    'azure-schemaregistry==1.0.0b3',
+    'azure-schemaregistry>=1.0.0,<2.0.0',
     'avro<2.0.0,==1.10.0',
```

Comment:
this is super nit, but I think we just need it to be `avro==1.10.0`, `<2.0.0` is unnecessary.

(also if you're gonna change it, don't forget the one in shared_requirements.txt)